main.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "null_resource" "ansible_provisioner" {
```

Comment:
I think we do not need a spacial resource for waiting, because we do it by special section in the "provision" playbook.
* Create a runtime inventorty with an ip address of a host
* Waite for target host is ready for ssh connection
```
---
- hosts: localhost
  vars:
    gather_facts: true
  tasks:
  - name: Add public ip addresses to an dynamic inventory
    add_host:
      name: "{{ host }}"
      groups: all

  - local_action: wait_for port=22 host="{{ host }}" search_regex=OpenSSH delay=10
```